waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/AuthController.kt

```diff
@@ -474,14 +477,14 @@ suspend fun verifyToken(token: String): Result<String> {
 suspend fun PipelineContext<Unit, ApplicationCall>.doLogin() {
     val reqBody = loginRequestJson.decodeFromString<AccountRequest>(call.receive())
     AccountsService.authenticate("", reqBody)
-        .onSuccess { // FIXME -> TENANT HERE
+        .onSuccess {
 
-            val tokenPayload = "{\"sub\":\"${it.id}\"}"
+            val iss = AuthKeys.issTokenClaim
+            val aud = AuthKeys.audTokenClaim
+            val tokenPayload = "{\"sub\":\"${it.id}\",\"iss\":\"$iss\",\"aud\":\"$aud\"}"
```

Comment:
since this is becoming a more complex object,
maybe have a serializable data class?